src/dinero.js

```diff
@@ -343,14 +410,21 @@ const Dinero = options => {
      * @example
      * // returns false
      * Dinero({ amount: 800 }).lessThan(Dinero({ amount: 500 }))
+     * @example
+     * // returns true
+     * Dinero({ amount: 5000, precision: 3 }).lessThan(Dinero({ amount: 800 }))
+     * @example
+     * // returns false
+     * Dinero({ amount: 800 }).lessThan(Dinero({ amount: 5000, precision: 3 }))
      *
      * @throws {TypeError} If `comparator` has a different currency.
      *
      * @return {Boolean}
      */
     lessThan(comparator) {
       assertSameCurrency.call(this, comparator)
-      return this.getAmount() < comparator.getAmount()
+      const comparators = Dinero.normalizePrecision([this, comparator])
```

Comment:
I'm not sure `normalizePrecision` is a good method name. While reading this code, I didn't understand what it would do, and I had to download the source to check for it. 

From your comments, it seems like you're setting to the highest precision, so maybe a better name will be `setToHighestPrecision` or something similar?

Reply:
I hear you, but I feel like "normalize" is a pretty common term in programming when it comes to making elements consistent, unified, "normalized". For example, `normalize.css` makes browsers render all elements more consistently.

`setToHighestPrecision` is more accurate according to what the method *does*, but it's not it's *purpose*. It's a good name for an internal method, I don't think it is for a public API (or there would need to be a `toHighestPrecision` and `toLowestPrecision`).

Does that make sense?

Reply:
@sarahdayan I hear you, and it makes sense. Just wondered if there's a better alternative. I'll keep my brain open for the next few days and let you know again if something comes up.

Reply:
How about `normalizeToHighestPrecusuon`? 

The name is slightly longer, but it speaks so much more about the function.

Reply:
It's the same problem. If you specify "to highest" then you're giving a clue about how it works under the hood instead of focusing on the purpose. You can't normalize in any other way without losing information, so there's no reason to have a `normalizeToLowestPrecision` method. Thus, giving this extra "to highest" precision doesn't seem necessary. That's what the docs are here for.

I'm all for self-explaining method names, but there needs to be a balance. Overly lengthy names that can be shortened thanks to good documentation without losing the gist of what it's doing are a sweet spot IMO.

Reply:
Sure thing. It's your call :)